[2024-10-03T08:02:36.715+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-03T08:02:36.777+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ETL_JsonData.unzip scheduled__2024-10-01T03:01:00+00:00 [queued]>
[2024-10-03T08:02:36.797+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ETL_JsonData.unzip scheduled__2024-10-01T03:01:00+00:00 [queued]>
[2024-10-03T08:02:36.799+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-10-03T08:02:36.948+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): unzip> on 2024-10-01 03:01:00+00:00
[2024-10-03T08:02:36.963+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ETL_JsonData', 'unzip', 'scheduled__2024-10-01T03:01:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/ETLProject/main.py', '--cfg-path', '/tmp/tmpx6k9u44s']
[2024-10-03T08:02:36.967+0000] {standard_task_runner.py:91} INFO - Job 36: Subtask unzip
[2024-10-03T08:02:36.969+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=819) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-03T08:02:36.972+0000] {standard_task_runner.py:63} INFO - Started process 831 to run task
[2024-10-03T08:02:37.054+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ETL_JsonData.unzip scheduled__2024-10-01T03:01:00+00:00 [running]> on host fb5d174c2391
[2024-10-03T08:02:37.201+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag_ETL_JsonData' AIRFLOW_CTX_TASK_ID='unzip' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T03:01:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-01T03:01:00+00:00'
[2024-10-03T08:02:37.203+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-03T08:02:37.226+0000] {logging_mixin.py:188} INFO - Lỗi khi giải nén file:[Errno 2] No such file or directory: 'C:/Users/adunh/Downloads/og_item_impression.20240830_/og_item_impression.20240830_1100_1.json.gz'
[2024-10-03T08:02:37.227+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-03T08:02:37.228+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-03T08:02:37.243+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_ETL_JsonData, task_id=unzip, run_id=scheduled__2024-10-01T03:01:00+00:00, execution_date=20241001T030100, start_date=20241003T080236, end_date=20241003T080237
[2024-10-03T08:02:37.310+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-03T08:02:37.331+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-03T08:02:37.335+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
